### PR TITLE
EF-1402-caliperEventLexileMeasureGraded for Spanish RI

### PR DESCRIPTION
**Pull request by mattihallim**
caliperEventLexileMeasureGraded for Spanish RI to be sent from Assessment Service

### CODE REVIEW

**Comment by mattihallim** [comment]
@odowdj

File: src/test/resources/fixtures/hmh-assessment-service/caliperEventSpanishRiGraded.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      "id":"urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
+      "type": "Assessment"
+    },
+    "count": 33,
+    "dateCreated": "2016-11-15T10:05:00.000Z",
+    "startedAtTime": "2016-11-15T10:05:00.000Z",
+    "endedAtTime": "2016-11-15T10:55:12.000Z"
+  },
+  "eventTime": "2016-11-15T10:57:06.000Z",
+  "edApp": "https://www.hmhco.com/AssessmentService",
```
Comment (odowdj) [inline src/test/resources/fixtures/hmh-assessment-service/caliperEventSpanishRiGraded.json]: Another Assessment Service exists in HMH - just wondering is there a more specific name for your application by any chance?

**Review by odowdj — APPROVED** [review]
lftm

**Comment by mattihallim** [comment]
@odowdj I dont see Merge button probably because "Only those with [write access](https://help.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests "  ?
Will you be able to look in to it ? I either need write-access or you may need to merge it ?

**Pull request merged**